docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: '3'
+services:
+  omega:
+    build: .
+    depends_on:
+    - redis
+    environment:
+    - HUBOT_SLACK_TOKEN=${SLACK_TOKEN}
+    restart: unless-stopped
```

Comment:
Why `unless-stopped` instead of `always`?

Reply:
`always` will always bring the container back up after a reboot (or docker restart), even if you stopped the container before the reboot.
https://blog.codeship.com/ensuring-containers-are-always-running-with-dockers-restart-policy/

Reply:
So I'm thinking, if you stopped the container, you probably don't want it back up just because the server rebooted....

Reply:
It's easier to remember a consistent behaviour of a server rather than a conditional one. So if and when a server restarts, this container come back up is easier to remember.

Also, I may stop the container so that it doesn't interfere with a task I'm doing, like patching the server, then restarting the server.

Therefore, I think it may be better to set it as `always`. What do you think?

Reply:
You have a point about stopping the containers before patching the server.

Anyway, it turns out running `docker-compose down` instead of `docker-compose stop` actually prevents the containers from upping again even with `restart: always`. So I'm going to set it as `always` instead.